release/pkg/assets_package_controller.go

```diff
@@ -44,7 +44,14 @@ func (r *ReleaseConfig) GetPackagesAssets() ([]Artifact, error) {
 	if err != nil {
 		return nil, errors.Cause(err)
 	}
-	sourceHelmURI, err := r.GetSourceHelmURI(repoName)
+	var sourceHelmURI string
```

Comment:
Is the value returned by `GetSourceHelmURI` incorrect for a production case? Is this the correct place to handle this branching? Or should the environment be passed to `GetShourceHelmURI`?

Reply:
The production role doesn't have access to call `GetSourceHelmURI`, and it's not needed so we want to skip that all together in that scenario.

Reply:
sounds good, fixed this to the way you mentioned.